daemon/daemon.go

```diff
@@ -1501,6 +1501,16 @@ func (daemon *Daemon) newBaseContainer(id string) *container.Container {
 	return container.NewBaseContainer(id, daemon.containerRoot(id))
 }
 
+// Reload reads configuration changes and modifies the
+// daemon according to those changes.
+// Currently, only the labels configuration is taken into account.
```

Comment:
The comment doesn't seem to match the code below, although I would have thought we'd need to tickle some additional business logic someplace to get the cluster advertising to update.  My understanding was going from no discovery configuration to an initial configuration should be relatively straight forward, but changing from one config to another while we're live is probably going to be much trickier and we probably want to require a restart for that sort of change to take effect.  Do you envision that sort of logic would live here or someplace else?